test/topology/conftest.py

```diff
@@ -150,7 +150,6 @@ async def manager_internal(event_loop, request):
     port = int(request.config.getoption('port'))
     ssl = bool(request.config.getoption('ssl'))
     manager_int = ManagerClient(request.config.getoption('manager_api'), port, ssl, cluster_con)
-    await manager_int.start()
```

Comment:
Why this change?

Reply:
Comment added in commit `HTTP client helper`
```
    While there, move aiohttp session setup back to constructors. When there
    were teardown issues it looked it could be caused by aiohttp session
    being created outside a coroutine. But this is proven not to be the case
    after recent fixes. So move it back to the ManagerClient constructor.
```

In latest version https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d

Reply:
I still don't understand why is it OK not to `start()` the ManagerClient at all. When is it started now?